Limits/python/Models.py

```diff
@@ -371,6 +376,271 @@ def build(self,ws,label):
+        sigmaSpline = ROOT.RooSpline1D(sigmaName, sigmaName, ws.var('MH'), len(masses), array('d',masses), array('d',sigmas))
+        aSpline =     ROOT.RooSpline1D(aName, aName, ws.var('MH'), len(masses), array('d',masses), array('d',a_s))
+        nSpline =     ROOT.RooSpline1D(nName, nName, ws.var('MH'), len(masses), array('d',masses), array('d',n_s))
+        # import
+        getattr(ws, "import")(meanSpline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(sigmaSpline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(aSpline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(nSpline, ROOT.RooFit.RecycleConflictNodes())
+        # build model
+        ws.factory("RooCBShape::{0}({1}, {2}, {3}, {4}, {5})".format(label,self.x,meanName,sigmaName,aName,nName))
+        self.params = [meanName,sigmaName,aName,nName]
+
+class DoubleCrystalBall(Model):
+
+    def __init__(self,name,**kwargs):
+        super(DoubleCrystalBall,self).__init__(name,**kwargs)
+        if not hasattr(ROOT,"DoubleCrystalBall"):
+            ROOT.gROOT.ProcessLine(".L /afs/cern.ch/work/k/ktos/public/Plotting/CMSSW_8_1_0/src/CombineLimits/Limits/macros/DoubleCrystalBall.cpp")        
+        
+
+    def build(self,ws,label):
+        logging.debug('Building {}'.format(label))
+        meanName = 'mean_{0}'.format(label)
+        sigmaName = 'sigma_{0}'.format(label)
+        a1Name = 'a1_{0}'.format(label)
+        n1Name = 'n1_{0}'.format(label)
+        a2Name = 'a2_{0}'.format(label)
+        n2Name = 'n2_{0}'.format(label)
+        mean  = self.kwargs.get('mean',  [1,0,1000])
+        sigma = self.kwargs.get('sigma', [1,0,100])
+        a1     = self.kwargs.get('a1', [1,0,100])
+        n1     = self.kwargs.get('n1', [1,0,100])
+        a2     = self.kwargs.get('a2', [1,0,100])
+        n2     = self.kwargs.get('n2', [1,0,100])    
+        # variables
+        ws.factory('{0}[{1}, {2}, {3}]'.format(meanName,*mean))
+        ws.factory('{0}[{1}, {2}, {3}]'.format(sigmaName,*sigma))
+        ws.factory('{0}[{1}, {2}, {3}]'.format(a1Name,*a1))
+        ws.factory('{0}[{1}, {2}, {3}]'.format(n1Name,*n1))
+        ws.factory('{0}[{1}, {2}, {3}]'.format(a2Name,*a2))
+        ws.factory('{0}[{1}, {2}, {3}]'.format(n2Name,*n2))
+
+        # build model
+        doubleCB = ROOT.DoubleCrystalBall(label, label, ws.arg(self.x), ws.arg(meanName), ws.arg(sigmaName), 
+                   ws.arg(a1Name), ws.arg(n1Name), ws.arg(a2Name), ws.arg(n2Name) )
+        self.wsimport(ws, doubleCB)
+        self.params = [meanName,sigmaName,a1Name,n1Name,a2Name,n2Name]
+
+class DoubleCrystalBallSpline(ModelSpline):
+
+    def __init__(self,name,**kwargs):
+        super(DoubleCrystalBallSpline,self).__init__(name,**kwargs)
+        if not hasattr(ROOT,"DoubleCrystalBall"):
+            ROOT.gROOT.ProcessLine(".L /afs/cern.ch/work/k/ktos/public/Plotting/CMSSW_8_1_0/src/CombineLimits/Limits/macros/DoubleCrystalBall.cpp")
```

Comment:
Can't do this, needs to be relative path:
`path = '{}/src/CombineLimits/Limits/macros/DoubleCrystalBall.cpp'.format(os.environ['CMSSW_BASE'])`